src/sc2api/sc2_unit.cc

```diff
@@ -14,6 +14,16 @@ bool Unit::IsBuildFinished() const {
     return build_progress >= 1.0f;
 }
 
+Tags GetTagsFromUnits(const Units& units) {
+    Tags tags;
+    for (const auto& unit : units) {
```

Comment:
There is no need in reference since sc2::Unit is a pointer which is a kind of int.